### PR TITLE
Chapter 13 final draft exercise

### PR DESCRIPTION
**Pull request by ian-flores**
This PR adds the draft for the final exercise for chapter 13. It includes two optional exercises which we can then delete if we think they are too out of scope.

### CODE REVIEW

File: chapters/13-package-release.Rmd [hunk 1]
```diff
@@ -10,4 +10,47 @@
...
+> `rhub::check(<OS>)`
+
+5. Browse previous checks
+> `rhub::list_package_checks()`
+
+B. Etiquette for using (mostly free) servers and external build systems like GitHub Actions and rhub (e.g. do not overuse them)
+
+1. Read the usage limits on GitHub Actions and rhub
+> - https://docs.github.com/en/actions/reference/usage-limits-billing-and-administration
+> - https://docs.r-hub.io/#regularly
```
Comment (cwickham) [inline chapters/13-package-release.Rmd:38]: I really like this!

File: chapters/13-package-release.Rmd [hunk 2]
```diff
@@ -10,4 +10,47 @@
...
+1(b). Run the GitHub Actions for the package, locally (optional)
+> https://github.com/nektos/act
+
+1(c). Run the rhub check locally (optional)
+> https://r-hub.github.io/rhub/articles/local-debugging.html
+
+C. Managing your versions, git tagging
+
+1. Upgrade version in the R package
+> In the DESCRIPTION file, modify the minor version number
```
Comment (cwickham) [inline chapters/13-package-release.Rmd:49]: What about `usethis::use_version()`, which also updates `NEWS`?

File: chapters/13-package-release.Rmd [hunk 3]
```diff
@@ -10,4 +10,47 @@
...
+
+1. Read the usage limits on GitHub Actions and rhub
+> - https://docs.github.com/en/actions/reference/usage-limits-billing-and-administration
+> - https://docs.r-hub.io/#regularly
+
+1(b). Run the GitHub Actions for the package, locally (optional)
+> https://github.com/nektos/act
+
+1(c). Run the rhub check locally (optional)
+> https://r-hub.github.io/rhub/articles/local-debugging.html
```
Comment (cwickham) [inline chapters/13-package-release.Rmd:44]: I'm not really familiar with these, are they hard to do?  Perhaps that is why they are optional?

File: chapters/13-package-release.Rmd [hunk 4]
```diff
@@ -10,4 +10,47 @@
...
+
+1. Upgrade version in the R package
+> In the DESCRIPTION file, modify the minor version number
+
+2. Build your package and commit
+> `devtools::build()`
+> `git commit`
+
+3. Tag your commit (version # is flexible)
+> `git tag -a v1.0 -m "Version Tag 1.0"`
```
Comment (cwickham) [inline chapters/13-package-release.Rmd:56]: Should we also get them to go through the release process on GitHub?

**Review by cwickham — APPROVED** [review]
This looks good.  Feels possibly a bit long, but only if the optional exercises are included.

File: chapters/13-package-release.Rmd (hunk 1 above)
Comment (lwjohnst86) [inline chapters/13-package-release.Rmd:38]:
```suggestion
> - https://docs.r-hub.io/#regularly
TODO: Add a question to assess whether they've read it?
```

File: chapters/13-package-release.Rmd (hunk 3 above)
Comment (lwjohnst86) [reply chapters/13-package-release.Rmd:44]: They aren't hard to do. They are all implemented really nicely in the rhub package. Maybe the only challenge might be working with docker, which rhub seems to do. Optional is a good choice for now, we can revisit later.

File: chapters/13-package-release.Rmd (hunk 2 above)
Comment (lwjohnst86) [inline chapters/13-package-release.Rmd:49]:
```suggestion
> In the DESCRIPTION file, use `usethis::use_version()` to modify the minor version number
```

File: chapters/13-package-release.Rmd (hunk 4 above)
Comment (lwjohnst86) [inline chapters/13-package-release.Rmd:56]: And also maybe use `use_release_issue()`, which gives a list of steps to take before releasing.

```suggestion
> `git tag -a v1.0 -m "Version Tag 1.0"`
TODO: Use use_release_issue() as well?
```

**Pull request merged**